syntax/javascript.vim

```diff
@@ -337,7 +337,7 @@ if version >= 508 || !exists("did_javascript_syn_inits")
   HiLink jsSwitchColon          Noise
   HiLink jsClassMethodType      Type
   HiLink jsObjectMethodType     Type
-  HiLink jsClassDefinition      jsFuncName
+  HiLink jsClassDefinition      Constant
```

Comment:
Ok.  I can definitely put this in an `after` for sure.  My take however would be to match everyone's 80% case and let them change it to something other than if they wanted.

Most people who use classes are going to want the name of the class highlighted as a constant (I'd assume).  If they want it highlighted as a function, then _they_ could go into an `after` and modify it to be so.

Basically it just boils down to how many people do you want to have to modify the syntax highlighting to make it like what they want.

But I'll remove it to get the rest of this merged and add it to my `after`. 👍🏻